normandy/recipes/templates/admin/field_recipe_list.html

```diff
@@ -0,0 +1,9 @@
+{% if recipes %}
+  <ul>
+    {% for recipe in recipes %}
+      <li><a href="{% url 'admin:recipes_recipe_change' recipe.id %}">{{ recipe.name }}</a></li>
+    {% endfor %}
+  </ul>
+{% else %}
+  <p>Not used in any recipes.</p>
```

Comment:
I think this should be "Not used in any active recipes". Either that, or this section should include both active and inactive recipes, and indicate that.